vignettes/EpiNow2.Rmd

```diff
@@ -0,0 +1,297 @@
+---
+title: "Getting started with EpiNow2"
+output:
+  rmarkdown::html_vignette:
+    toc: false
+    number_sections: false
+bibliography: library.bib
+csl: https://raw.githubusercontent.com/citation-style-language/styles/master/apa-numeric-superscript-brackets.csl
+vignette: >
+  %\VignetteIndexEntry{Getting started with EpiNow2}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+## Quick start
+
+In the following section we give an overview of the simple use case for `epinow()` and `regional_epinow()`.
+
+The first step to using the package is to load it as follows.
+
+
+```r
+library(EpiNow2)
+```
+
+### Reporting delays, incubation period and generation time
+
+Distributions can be supplied in two ways. First, by supplying delay data to `estimate_delay()`, where a subsampled bootstrapped lognormal will be fit to account for uncertainty in the observed data without being biased by changes in incidence (see `?EpiNow2::estimate_delay()`).
+
+Second, by specifying predetermined delays with uncertainty using `dist_spec()`. An arbitrary number of delay distributions are supported in `dist_spec()` with the most common use case likely to be an incubation period followed by a reporting delay (see `?EpiNow2::dist_spec()`).
+
+For example if data on the delay between onset and infection was available we could fit a distribution to it, using `estimate_delay()`, with appropriate uncertainty as follows (note this is a synthetic example),
+
+```r
+reporting_delay <- estimate_delay(
+  rlnorm(1000, log(2), 1),
+  max_value = 14, bootstraps = 1
+)
+```
+
+If data was not available we could instead specify an informed estimate of the likely delay
+using `dist_spec()`. To demonstrate, we choose a lognormal distribution with mean 2, standard deviation 1 (both with some uncertainty) and a maximum of 10. *This is just an example and unlikely to apply in any particular use case*.
+
+
+```r
+reporting_delay <- dist_spec(
+  mean = convert_to_logmean(2, 1),
+  mean_sd = 0.1,
+  sd = convert_to_logsd(2, 1),
+  sd_sd = 0.1,
+  max = 10,
+  dist = "lognormal"
+)
+```
+
+```
+## Warning in dist_spec(mean = convert_to_logmean(2, 1), mean_sd = 0.1, sd = convert_to_logsd(2, :
```

Comment:
fix this?

Reply:
Fixed in 23a6ad1165d599309d8cfc4ea4ce947302b9f234.